PKGBUILD

```diff
@@ -18,4 +18,4 @@ package() {
     install -m 755 -D temp "${pkgdir}/usr/local/bin/\$" 
```

Comment:
```suggestion
    install -m 755 -D dollarskip "${pkgdir}/usr/local/bin/\$" 
```

Reply:
I meant to change that with the features of the new makefile. See latest commit.